contracts/Config.sol

```diff
@@ -9,7 +9,7 @@ contract Config is IConfig, OwnableUpgradeable {
 
     address private crankAddress;
     address private feeReceiver;
-    uint8 private validForDays;
+    uint256 private validForSeconds;
```

Comment:
why bother making this private if you have an external function for returning this very value? it just adds extra bloat to the contract

Reply:
updated